cime_config/tests.py

```diff
@@ -166,11 +166,11 @@
             "SMS_Ld2.ne30_oECv3.BGCEXP_CNTL_CNPRDCTC_1850.elm-bgcexp",
             "SMS_D_Ld1.T62_oEC60to30v3.DTESTM",
             "SMS_D_Ld1.ne30pg2_r05_EC30to60E2r2.A_WCYCL1850S_CMIP6",
-            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX",
-            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_use_VT",
+            "ERP_Ln9.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_fixed_subcycle",
+            "ERS_Ln9.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_use_VT",
             "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_use_ESMT",
```

Comment:
These two are erroring out on Theta (Cori-KNL too) with `forrtl: severe (154): array index out of bounds`:
```
ERS_Ln9.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_use_VT
ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_use_ESMT
```

Reply:
That's odd. The "use_ESMT" test was having issues on Cori, but I thought I fixed it by putting the "96x1" back in. I need to do some digging to figure this out.

Reply:
I just ran some tests on Cori and the "use_VT" test passes with 2 threads, but fails with 1... Not sure what to make of that.

Reply:
Ok, I've identified the issue. It was a tricky to track down, but both tests were actually broken by a previous PR that updated the YAKL FFT routines. I'm going to create an issue, but in the mean time those tests will need to stay broken until we can create a new PR to update the FFT calls.